addons/block_code/serialization/block_script_serialization.gd

```diff
@@ -182,6 +189,17 @@ func _get_parent_node_property_info(property_name: String) -> Dictionary:
 	return {}
 
 
+func _get_resource_block_definition(block_name: String, file_path: String) -> BlockDefinition:
+	if not block_name.begins_with("get_resource_"):
+		return null
+
+	if file_path == "":
+		return null
+
+	var file_name = file_path.get_file()
```

Comment:
Is there a chance that this string doesn't have a valid file? Eg. what would happen if the user removes the file from the filesystem? In that case what does `get_file()` return?

Reply:
`get_file()` simply returns the file name part of the "string".  Fixed in the new commit which checks with `FileAccess.file_exists()` and `ResourceLoader.exists()`.